vue-frontend/src/components/home-new/ContributionSection.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="tw-relative">
+    <div class="tw-container tw-mt-2 md:tw-mt-4">
+      <div
+        class="tw-sticky tw-w-full tw-h-[30vh] sm:tw-h-[25vh] lg:tw-h-[35vh] tw-z-[1] tw-top-[6rem] tw-bg-white tw-pt-12 tw-text-center"
+      >
+        <h1
+          class="tw-font-inter-bold tw-text-black-core/[.87] tw-text-[1.875rem] tw-leading-[2.5rem] md:tw-text-[3.4375rem] md:tw-leading-[5.5625rem]"
+        >
+          We are here to Contribute
+        </h1>
+        <span
+          class="tw-font-inter-medium tw-text-black-core/[0.6] tw-text-[1rem] tw-leading-[1.375rem] md:tw-text-[1.5rem] md:tw-leading-[2.5625rem]"
+          >and that starts with the site itself,
+          <a
+            class="tw-relative tw-bg-gradient-[1deg] tw-pb-[5px] v2-canopas-gradient-text tw-font-inter-medium gradient-underline tw-no-underline after:tw-absolute after:tw-w-full after:tw-h-[2px] after:tw-bg-gradient-to-l tw-to-pink-300 tw-from-orange-300 after:tw-bottom-0 after:tw-left-0"
+            :href="websiteOpenSourceUrl"
+            target="_blank"
+            @click.native="$gtag.event('tap_canopas_website_github')"
+            >canopas is open source!</a
+          ></span
+        >
+      </div>
+
+      <div
+        v-for="(contribution, index) in contributions"
+        :key="contribution"
+        class="tw-flex tw-flex-row tw-justify-center tw-mt-14 sm:tw-mt-4"
+        :class="index % 2 != 0 ? 'tw-flex-row-reverse' : ''"
+      >
+        <div
+          class="tw-w-[13.625rem] md:tw-w-fit tw-h-[26.875rem] md:tw-h-fit tw-shadow-[2px_2px_10px_rgba(0,0,0,0.1)]"
+        >
+          <img
+            :src="contribution.video"
+            :alt="contribution.title + `image`"
+            class="tw-w-full tw-h-full"
+          />
+        </div>
+
+        <div
+          class="tw-flex tw-flex-col tw-basis-[40%] md:tw-basis-[55%] lg:tw-basis-[35%] tw-my-auto tw-bg-[#FFFFFF] tw-p-2.5 sm:tw-p-4 v2-title-3-text tw-shadow-[0_0px_10px_rgba(0,0,0,0.1)]"
```

Comment:
What is the use of the `v2-title-3-text` class as we are adding font size classes separately?